packages/itwinui-css/package.json

```diff
@@ -49,10 +49,11 @@
     "build": "yarn clean && yarn build:css && node ../shared/copyrightLinter.js --fix css/* && yarn build:html",
     "build:css": "node ./scripts/generateCss.js src css",
     "build:html": "vite build backstop/tests --config vite.config.ts",
-    "build:watch:scss": "sass --watch src/all.scss css/all.css",
+    "build:watch:scss": "sass --watch src/all.scss:css/all.css src/global.scss:css/global.css",
     "createComponent": "node ./scripts/createComponent.js",
     "clean": "rimraf ./css ./scss ./backstop/results/bitmaps_test",
-    "dev": "concurrently -n scss,html -k \"yarn build:watch:scss\" \"yarn vite backstop/tests --config vite.config.ts\"",
+    "dev:setup": "sass src/anchor/classes.scss:css/anchor.css src/all.scss:css/all.css src/global.scss:css/global.css",
+    "dev": "yarn dev:setup && concurrently -n scss,html -k \"yarn build:watch:scss\" \"yarn vite backstop/tests --config vite.config.ts\"",
```

Comment:
with these changes, there are too many places to keep track of. in the future i we need to add a button to index.html, there is a good chance we might forget to update one of these places.

it might just be simpler to change `index.html` to import `all.css` instead of `global.css` and `anchor.css`. the downside is that it will download more css than is needed, but it's not too much and is an acceptable tradeoff to me for maintainability.

Reply:
Sounds good, `index.html` now imports `all.css` and I removed the `dev:setup` script.

I initially did not do it this way to reduce the size of the downloaded css. Just for context, these are the values I got in `Inspector -> Network` when `index.html` imports:
* only `anchor.css`:
![image](https://user-images.githubusercontent.com/45748283/208465596-86ab9c3c-8fa1-4582-a8c0-92fd3661c440.png)
* `all.css`: 
![image](https://user-images.githubusercontent.com/45748283/208465787-e8c9b2cc-6b33-44c9-b364-23ae8e798526.png)

Reply:
is that filtered by css? doesn't look right

i expect around <35kB with all.css and <10kB with anchor.css

Reply:
I did not add a CSS filter. When filtering by CSS, it showed `0B` and no items in the list even after a hard refresh. Not sure why that is.

![image](https://user-images.githubusercontent.com/45748283/208470195-cf8f7156-7387-4281-92ce-c739b0fe0733.png)

Reply:
you need to check the prod build. run `yarn build` followed by `yarn preview` inside `packages/itwinui-css`

Reply:
It still doesn't show the specific css files when applying the CSS filter. But these are the new values I got when `index.html` imports:

* `all.css`
![image](https://user-images.githubusercontent.com/45748283/208495831-1de161b4-c065-4c86-a7a9-95ca0533295d.png)

* `global.css` and `anchor.css`
![image](https://user-images.githubusercontent.com/45748283/208496316-a6e54915-178b-4111-a396-f1e6da7de49b.png)

So now the size difference between the two imports is smaller, around `28kB transferred | 238 kB resources`

Reply:
ah, that's because the css is getting inlined into a `<style>` tag instead of a separate css file. also the total transferred size might be bloated by any browser extensions you have installed.

so you need to look at the size of the initial document.

| with `global`+`anchor` | with `all` |
| --- | --- |
| ![6.3kB](https://user-images.githubusercontent.com/9084735/208532098-df0fff0c-0019-43ee-bc9b-81e75abe757d.png) | ![34.8kB](https://user-images.githubusercontent.com/9084735/208531919-62ce7f25-7d48-4f60-aa3b-f9668e085b6f.png) |